src/Perl6/Optimizer.nqp

```diff
@@ -20,7 +20,7 @@ my class Symbols {
     # Some interesting scopes.
     has $!GLOBALish;
     has $!UNIT;
-    has $!SETTING;
+    # has $!SETTING;
```

Comment:
Why comment out when it is not needed anymore?

Reply:
Temporary state of things. Explanation is below.